projects/ebv-analyzer-app/src/app/ebv-selector/ebv-selector.component.html

```diff
@@ -122,23 +122,18 @@
 
     <mat-spinner *ngIf="loading$ | async"></mat-spinner>
 
-    <div *ngIf="false">
-        <mat-divider></mat-divider>
-
-        <!-- <div fxLayout="row" fxLayoutAlign="space-between start">
-            <wave-ebv-country-selector fxFlex=""></wave-ebv-country-selector>
-            <span fxFlex="1rem"></span>
-            <button mat-raised-button color="primary" [disabled]="isPlotButtonDisabled$ | async" (click)="plot()">Calculate</button>
-        </div> -->
+    <mat-divider></mat-divider>
 
-        <!-- TODO: implement -->
-        <!-- <wave-ebv-indicator-plot *ngIf="plotSettings"
-                                 [data]="plotSettings.data$"
-                                 [xLimits]="plotSettings.xLimits"
-                                 [yLabel]="plotSettings.yLabel"
-                                 [yLimits]="plotSettings.yLimits"></wave-ebv-indicator-plot> -->
+    <div fxLayout="row" fxLayoutAlign="space-between start">
+        <wave-ebv-country-selector fxFlex=""></wave-ebv-country-selector>
+        <span fxFlex="1rem"></span>
+        <button mat-raised-button color="primary" [disabled]="isPlotButtonDisabled$ | async" (click)="plot()">Calculate</button>
```

Comment:
In the current EBV Analyzer, we only show this after we selected an entity. I guess it makes sense, since we only then allow some computation.

Reply:
You can also already click calculate and nothing happens.

Reply:
268112d718dfd89b7fa17a05dbfe6748221c3b48